1228-public_page/1530-files_with_administrative_agreement_is_not_visible.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Files with Administrative Agreement is not visible
+
+    @1530
+    Scenario: Files with Administrative Agreement is not visible
+        Given a User views the Landing Page for a Publication
```

Comment:
Who is User in this context (*any* User, Anonymous, Creator, etc.?)

Reply:
This applies for _any_ user. But I can put in the word `any`.